### PR TITLE
REM-1018: Add consensus parser

### PR DESCRIPTION
**Pull request by DeV1doR**

### CODE REVIEW

**Comment by DeV1doR** [comment]
Waiting for implementation

File: remme/clients/block_info.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import logging
+import asyncio
 
 from remme.clients.basic import BasicClient
 from remme.protos.block_info_pb2 import BlockInfo, BlockInfoConfig
+# TODO: Uncomment when proto will be available
```
Comment (eugene-babichenko) [inline remme/clients/block_info.py]: Just update the `protos` submodule, it's already here.

Comment (DeV1doR) [reply remme/clients/block_info.py]: Will do

**Comment by DeV1doR** [comment]
Got expected result:
```
         "consensus" : {
            "previous_cert_votes" : [
               {
                  "message_signature" : "I6c65piJgXoSp4txqFczBwrrf44LDyhngVEPl9pSjs4vVpLjX9fkOJs+fwgAREb45gwkWC9/hVD2mUo6SRnBaA==",
                  "message" : {
                     "type" : "CERT_VOTE",
                     "payload" : {
                        "voter_id" : "AkAJPM5XAwt65rkDBuFwmhpUc2J3bk0Vx7ZevFXi3oPM",
                        "proposal_id" : "dGF7HWtpTNyknLRHJa5RolX/XX1Ud4AaomCqZRnLlUNitxybCOCYMHwdjdiNFeCyheIK6xKavWXrGUg/BxUxvA==",
                        "period" : "0"
                     }
                  }
               }
            ]
         },
```

**Pull request merged**